README.md

```diff
@@ -52,23 +51,47 @@ Topical Calcipotriene Treatment for Breast Cancer Immunoprevention, France
 
 - [ ] Clone this repository (do **not** fork it)
 - [ ] Implement the features step-by-step (your commit history should be clear to follow)
-- [ ] Document your architecture and design choices along the way
-- [ ] Provide instructions on how we can run your code
+- [ ] Document & explain your architecture and design choices along the way
+- [ ] Provide clear instructions on how we can run your code
 - [ ] Publish it on GitHub (or equivalent)
-- [ ] Send us the link and tell us approximately how much time you spent on this assignment
+- [ ] Send us the link, along with an estimate of how much time you spent on this assignment
 
 ## Guidelines
 
+### Starting pack
+To get you started quicker, we setup a typescript monorepo with an API and a CLI.
+
+### Choose your stack
+We do provide a starting pack with a typescript monorepo, but you are free to use any language or framework you are comfortable with.
+You may choose any architectural and design approach, as long as you can explain your decisions.
+You are encouraged to make good use of open-source code.
+
+Example stack (not limited to): http server exposing REST endpoint that serves json payloads.
+
+### Request for clarification
+If anything is unclear, please:
+- contact us, we’ll be happy to clarify
+- or make reasonable assumptions and document them
```

Comment:
I'd suggest this only (and not "cotnact us..."), otherwise there is a risk to add friction and hazards on the process.

The test is clear enough, and it's quite standard to ask the candidate to _make reasonable assumptions and document them_